lib/reek/ast/object_refs.rb

```diff
@@ -1,33 +1,30 @@
 module Reek
-  module AST
+  module AST  # :nodoc:
```

Comment:
We could put this rubocop exception in a rubocop config file so that it doesn't pollute our source code, couldn't we?